tests/unit/Actions.spec.ts

```diff
@@ -180,7 +180,7 @@ describe('Emits error event if NR validation fails in file and pay', () => {
   })
 })
 
-describe('Actions component - Filing Functionality', () => {
+describe.only('Actions component - Filing Functionality', () => {
```

Comment:
Don't commit this.

Reply:
You mean i can't get all the unit tests to pass by not running them!? Fine....

Will fix this in next commit 😆